imports/api/topics/votings/votings.js

```diff
@@ -237,14 +237,16 @@ Topics.categoryHelpers('vote', {
   },
   voteSummaryDisplay() {
     if (!this.voteSummary) return [];   // Results come down in a different sub, so it might not be there just yet
-    return Object.keys(this.voteSummary).map(key => {
+    const voteSummarydata = Object.keys(this.voteSummary).map(key => {
       const choice = this.vote.choices[key];
-      const votingUnits = this.voteSummary[key];
+      const votingUnits = this.voteSummary[key].round(0);
       const votingShare = this.unitsToShare(this.voteSummary[key]);
-      const percentOfTotal = votingShare.toNumber() * 100;
-      const percentOfVotes = (votingUnits / this.voteParticipation.units) * 100;
+      const percentOfTotal = (votingShare.toNumber() * 100).round(2);
```

Comment:
Van egy olyan altalanos szabaly, hogy kerekiteni szamolas kozben nem szoktunk, csak a folyamat legvegen, megjeleniteskor. Mert ha tobbszor kerekitek egy tobblepcsos szamitasi folyamat soran ,akkor a kerekitesi vesztesegek nemcsak osszeadodnak, de akár ossze is szorzodhatnak, es nagyra nohetnek egyutt